content/community/not-cucumber.md

```diff
@@ -19,8 +19,15 @@ Note that if you are using a newer version of Cucumber, the plugin you are using
 ## IntelliJ
 {{% text "javascript,ruby" %}}IntelliJ is a Java IDE.{{% /text %}}
 {{% block "java,kotlin" %}}
-If you are having issues with the "Cucumber for Java" plugin, please check [YouTrack](https://youtrack.jetbrains.com) to see if your issue has been reported yet (and vote for it!).
-If not, you can create an issue yourself.
+JetBrains are the creators of IntelliJ, and maintain the "Cucumber for Java" plugin, please check [YouTrack](https://youtrack.jetbrains.com) to see if your issue has been reported yet (and vote for it!).
+If not, you can create an issue yourself. You need to
+
+IntelliJ and Cucumber v1.x
+:  For "Cucumber java" check that you are using the "Program arguments:" of ```--plugin org.jetbrains.plugins.cucumber.java.run.CucumberJvmSMFormatter```
+
```

Comment:
As mentioned by @luke-hill I'm not sure we should be mentioning using v1 anymore as it is very old. If we do, we should also mention v2 and v3 in my opinion. (I'm not sure as of when / which version the output while executing changed). 
As the info as proposed might be outdated at some, how about something more generic, like: "The plugin follows the development of Cucumber-jvm and might be out of sync. When that happens, you might need to remove the `CucumberJvmXSMFormatter` from your program arguments." 

What do you think @nhojpatrick @mpkorstanje ?

Reply:
@nhojpatrick Would you be willing to make it more generic? If not, net me know and I could do it myself.

Reply:
@mlvandijk @mpkorstanje @luke-hill just done and update... think it can be clean up but it is more generic and more true.